src/buildMathML.js

```diff
@@ -471,6 +493,65 @@ groupTypes.underline = function(group, options) {
     return node;
 };
 
+groupTypes.accentunder = function(group, options) {
+    const accentNode = stretchy.mathMLnode(group.value.label);
+    const node = new mathMLTree.MathNode(
+        "munder",
+        [buildGroup(group.value.body, options), accentNode]
+    );
+    node.setAttribute("accentunder", "true");
```

Comment:
Maybe I should have read this comment before I got pedantic in the previous one.